letsencrypt/renewer.py

```diff
@@ -0,0 +1,117 @@
+"""Renewer tool to handle autorenewal and autodeployment of renewed
+certs within lineages of successor certificates, according to
+configuration."""
+
+# TODO: sanity checking consistency, validity, freshness?
+
+# TODO: call new installer API to restart servers after deployment
+
+# TODO: when renewing or deploying, update config file to
+#       memorialize the fact that it happened
+
+import configobj
+import os
+
+from letsencrypt import configuration
+from letsencrypt import client
+from letsencrypt import crypto_util
+from letsencrypt import notify
+from letsencrypt import storage
+from letsencrypt.plugins import disco as plugins_disco
+
+DEFAULTS = configobj.ConfigObj("renewal.conf")
```

Comment:
What is that `renewal.conf` referring to? Where do you actually load the configuration file?